src/main/java/gov/nasa/pds/tools/validate/rule/pds4/TableDataContentValidationRule.java

```diff
@@ -133,11 +174,136 @@ public boolean isApplicable(String location) {
     }
     return isApplicable;
   }
+
+  private void validateAsciiStringFieldsFormat(List<String> fieldFormatList, List<String> fieldTypeList, List<String> fieldNumberList) {
+    // ASCII String related fields should not have '+' in the format.
+
+    int indexToList = 0;
+    // Loop through all elements fieldFormatList.  If field is in ASCII_STRING_TYPE_LIST, check if prohibited '+' is in the format.
+    for (String oneFieldFormat : fieldFormatList) {
+        //LOG.debug("validateAsciiStringFieldsFormat:indexToList,oneFieldFormat,fieldTypeList.get(indexToList)  {},{},{}",indexToList,oneFieldFormat,fieldTypeList.get(indexToList));
+        // Check for prohibited plus symbol '+' in format field : %+8s
+        if (this.ASCII_STRING_TYPE_LIST.contains(fieldTypeList.get(indexToList).toUpperCase())) {
```

Comment:
`ASCII_STRING_TYPE_LIST` is a static variable, so we shouldn't be using `this.` here.